cg/server/api.py

```diff
@@ -364,21 +364,35 @@ def get_sequencing_metrics(flow_cell_name: str):
     """Return sample lane sequencing metrics for a flow cell."""
     if not flow_cell_name:
         return jsonify({"error": "Invalid or missing flow cell id"}), HTTPStatus.BAD_REQUEST
-    sequencing_metrics: list[SampleLaneSequencingMetrics] = (
-        db.get_sample_lane_sequencing_metrics_by_flow_cell_name(flow_cell_name)
+    sequencing_metrics: list[IlluminaSampleSequencingMetrics] = (
+        db.get_illumina_sequencing_run_by_device_internal_id(flow_cell_name).sample_metrics
     )
     if not sequencing_metrics:
         return (
             jsonify({"error": f"Sequencing metrics not found for flow cell {flow_cell_name}."}),
             HTTPStatus.NOT_FOUND,
         )
-    metrics_dtos: list[SequencingMetricsRequest] = [
-        SequencingMetricsRequest.model_validate(metric, from_attributes=True)
-        for metric in sequencing_metrics
-    ]
+    metrics_dtos: list[SequencingMetricsRequest] = parse_metrics_into_request(sequencing_metrics)
     return jsonify([metric.model_dump() for metric in metrics_dtos])
 
 
+def parse_metrics_into_request(
+    metrics: list[IlluminaSampleSequencingMetrics],
+) -> list[SequencingMetricsRequest]:
+    """Parse  sequencing metrics into a request."""
```

Comment:
extra space in docstring
```suggestion
    """Parse sequencing metrics into a request."""
```